modules/instrument_list/php/instrument_list.class.inc

```diff
@@ -296,6 +296,18 @@ class Instrument_List extends \NDB_Menu_Filter
 
         $this->tpl_data['display']
             = array_merge($this->candidate->getData(), $this->timePoint->getData());
+        // DoB to Derived Age
```

Comment:
$dob_age=null;
        // DoB to Derived Age
        $dob       = $this->candidate->getData();
      if ($dob['DoB'] !=null && isset($dob['DoB']) ) {
        $dob_year  = abs((date('Y', strtotime($dob['DoB'])) - date('Y')));
        $dob_month = abs((date('m', strtotime($dob['DoB'])) - date('m')));
           $dob_age   = $dob_year . " y / " . $dob_month . " m";
      }       
              $this->tpl_data['dob_age'] = $dob_age;

Reply:
same to edc